src/main/resources/db/changelog/19-create-and-fill-position-status-table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE position_status (
+    id BIGSERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL UNIQUE
+);
+
+INSERT INTO position_status (name) VALUES ('Opened');
+INSERT INTO position_status (name) VALUES ('Filled from the platform');
+INSERT INTO position_status (name) VALUES ('Filled outside of the platform');
+INSERT INTO position_status (name) VALUES ('Canceled');
+
+ALTER TABLE positions ADD COLUMN status_id BIGINT;
+
```

Comment:
We should also DROP this row from positions table: **is_active BOOLEAN NOT NULL DEFAULT TRUE,**